LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -363,29 +405,29 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <source>
                                     <listBibl type="catalogue">
                                         <bibl>
-                                            <ptr target="bm:Wright1877BM"/>
-                                            <citedRange unit="page">19b-20a</citedRange>
+                                            <ptr target="bm:Wright1877BM"/><citedRange unit="page">19b-20a</citedRange>
                                         </bibl>
                                     </listBibl>
                                     <listBibl type="secondary">
                                         <bibl><ptr target="bm:Knibb2015Ezekiel"/><citedRange unit="page">12-13</citedRange></bibl>
                                         <bibl><ptr target="bm:Uhlig1988Palaographie"/><citedRange unit="page">383-386</citedRange></bibl>
                                         <bibl><ptr target="bm:Knibb1980LivesProphets1"/><citedRange unit="page">198-199</citedRange></bibl>
+                                        <bibl><ptr target="bm:Wright1870Magdala"/><citedRange unit="page">600</citedRange></bibl>
+                                        <bibl><ptr target="bm:Charles1900AscensionIsaiah"/><citedRange unit="page">XV</citedRange></bibl>
+                                        <bibl><ptr target="bm:PerroneNorelli1995Isaia"/></bibl>
+                                        <bibl><ptr target="bm:Loefgren1930Jonanah"/></bibl>
+                                        <bibl><ptr target="bm:Löfgren1927Daniel"/><citedRange unit="page">XV, XXV-XXVI</citedRange></bibl>
```

Comment:
The tag should be corrected in Zotero and here then )

Reply:
I see, you or somebody else corrected it in Zotero, already. I changed it here in the record accordingly.